hw1/02-palindrome.js

```diff
@@ -23,18 +23,18 @@ function isAlphabetsPresent(value) {
   return regExp.test(value);
 }
 
-function isPalindrome(value) {
-  const array = value.toString().split("");
+let isPalindrome = function checkIfValueIsPalindrome(value) {
+  const array = value.split("");
```

Comment:
Looks good except it doesn't quite fully follow the airbnb style guide. Airbnb uses single quotes. I found this article helpful while setting up eslint to use the airbnb style: https://medium.com/@Tunmise/set-up-eslint-with-airbnb-style-guide-in-5-minutes-d7b4cc5707f8

Good job!